recipes/metaphlan2/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "metaphlan2" %}
-{% set version = "2.9.22" %}
-{% set sha256 = "e95743a6a18dab92abee98ba48cb87a9d3f85a9cdb0fa773e768453c1cfcc521" %}
+{% set version = "2.9.5" %}
```

Comment:
Isn't this an older release?

Reply:
No, we have changed the versioning number

Reply:
Just keep in mind that no one will get 2.9.5 unless they explicitly request it, since 2.9.22 will always be seen as a higher release number.